packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/__brick__/lib/feature_onboarding/ui_components/register_form.dart

```diff
@@ -0,0 +1,136 @@
+{{> licence.dart }}
+
+import 'package:flutter/material.dart';
+import 'package:flutter_rx_bloc/flutter_rx_bloc.dart';
+import 'package:flutter_rx_bloc/rx_form.dart';
+import 'package:provider/provider.dart';
+import 'package:widget_toolkit/widget_toolkit.dart' hide ErrorModel;
+
+import '../../app_extensions.dart';
+import '../../base/common_ui_components/app_error_modal_widget.dart';
+import '../../base/extensions/async_snapshot_extensions.dart';
+import '../../base/extensions/error_model_field_translations.dart';
+import '../blocs/onboarding_bloc.dart';
+
+class RegisterForm extends StatefulWidget {
+  const RegisterForm({
+    required this.title,
+    super.key,
+  });
+
+  final String title;
+
+  @override
+  State<RegisterForm> createState() => _RegisterFormState();
+}
+
+class _RegisterFormState extends State<RegisterForm> {
+  final _emailFocusNode = FocusNode(debugLabel: 'emailFocus');
+  final _passwordFocusNode = FocusNode(debugLabel: 'passwordFocus');
+
+  /// Get all focus nodes of the form
+  List<FocusNode> get _focusNodes => [
+        _emailFocusNode,
+        _passwordFocusNode,
+      ];
+
+  @override
+  void dispose() {
+// Since the focus nodes are long living object, they should be disposed.
+    for (var focusNode in _focusNodes) {
+      focusNode.dispose();
+    }
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) => Column(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: [
+          Text(
+            widget.title,
+            style: context.designSystem.typography.h3Reg14,
+            textAlign: TextAlign.center,
+          ),
+          RxTextFormFieldBuilder<OnboardingBlocType>(
+            state: (bloc) => bloc.states.email.translate(context),
+            showErrorState: (bloc) => bloc.states.showFieldErrors,
+            onChanged: (bloc, value) => bloc.events.setEmail(value),
+            builder: (fieldState) => _buildEmailField(
+              fieldState,
+              context,
+            ),
+          ),
+          SizedBox(height: context.designSystem.spacing.xs1),
```

Comment:
Please make sure the latest Flutter capabilities are used instead of the old-fashion SizedBox approach

Reply:
Could you elaborate on what you mean? Do you mean to use the `spacing` property

Reply:
yes

Reply:
Done